conn.go

```diff
@@ -468,6 +475,19 @@ func (s *startupCoordinator) options(ctx context.Context) error {
 func (s *startupCoordinator) startup(ctx context.Context) error {
 	m := map[string]string{
 		"CQL_VERSION": s.conn.cfg.CQLVersion,
+		"DRIVER_NAME": driverName,
+	}
+
+	buildInfo, ok := debug.ReadBuildInfo()
```

Comment:
Number of dependencies might be huge, and BuildInfo is constant across each call. `startupCoordinator` is created on each new connection, so i would extract version into global unexported variable, and read it once upon module initialization using `init` function, as I suggested in the comment.

Reply:
Done.